modules/core/include/opencv2/core/cvdef.h

```diff
@@ -470,6 +470,17 @@ Cv64suf;
 #define CV_WRAP_PHANTOM(phantom_header)
 #define CV_WRAP_DEFAULT(val)
 
+/* To suppress warnings/errors from UBSan. */
+#if defined(__has_attribute)
+#  if __has_attribute(no_sanitize)
+#    define CV_NO_UBSAN __attribute__((no_sanitize("undefined")))
```

Comment:
We don't need this in the current patch.

Reply:
I thought it might be useful in the future, but I'll drop the commit.